docs/design/proposed-template-refactor.md

```diff
@@ -1,6 +1,6 @@
-# Acs-engine
+# AKS-Engine
 
-The Azure Container Service Engine (aks-engine) is a command line tool that generates ARM (Azure Resource Manager) templates in order for one to deploy container-based clusters (like Kubernetes , DCOS, Openshift, Docker swarm) on the Azure platform.
+AKS-Engine is a command line tool that generates ARM (Azure Resource Manager) templates in order for one to deploy container-based clusters (like Kubernetes , DCOS, Openshift, Docker swarm) on the Azure platform.
```

Comment:
should we remove `DCOS, Openshift, Docker swarm` ?

Reply:
Yes, because we will be removing that non-ACS-dependent stuff that implements those orchestrators.

Reply:
> we will be removing that non-ACS-dependent stuff that implements those orchestrators

See #92.